content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+[*] Nmap: |     References:
+[*] Nmap: |       https://technet.microsoft.com/en-us/library/security/ms17-010.aspx
+[*] Nmap: |       https://blogs.technet.microsoft.com/msrc/2017/05/12/customer-guidance-for-wannacrypt-attacks/
+[*] Nmap: |_      https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-0143
+[*] Nmap: Nmap done: 1 IP address (1 host up) scanned in 15.51 seconds
+```
+
+Given that I had a potential attack I decided to do a quick recap of the services on the host to make sure I didn't miss anything and saw that I'd overlooked RDP listening, which ended up being useful later in the attack.
+
+```
+msf5 > services 
+Services
+========
+
+host          port   proto  name           state  info
+----          ----   -----  ----           -----  ----
+10.10.201.83  135    tcp    msrpc          open   Microsoft Windows RPC
+10.10.201.83  139    tcp    netbios-ssn    open   Microsoft Windows netbios-ssn
+10.10.201.83  445    tcp    microsoft-ds   open   Windows 7 Professional 7601 Service Pack 1 microsoft-ds workgroup: WORKGROUP
+10.10.201.83  3389   tcp    ms-wbt-server  open   
+10.10.201.83  49152  tcp    unknown        open   Microsoft Windows RPC
+10.10.201.83  49153  tcp    unknown        open   Microsoft Windows RPC
+10.10.201.83  49154  tcp    unknown        open   Microsoft Windows RPC
+10.10.201.83  49158  tcp    unknown        open   Microsoft Windows RPC
+10.10.201.83  49160  tcp    unknown        open   Microsoft Windows RPC
+```
+
+### Preparing an attack
+First step was seeing if there was any exploit available for `ms17-010`
+
+```
+msf5 > search ms17-010
+
+Matching Modules
+================
+
+   #  Name                                           Disclosure Date  Rank     Check  Description
+   -  ----                                           ---------------  ----     -----  -----------
+   0  auxiliary/admin/smb/ms17_010_command           2017-03-14       normal   No     MS17-010 EternalRomance/EternalSynergy/EternalChampion SMB Remote Windows Command Execution
+   1  auxiliary/scanner/smb/smb_ms17_010                              normal   No     MS17-010 SMB RCE Detection
+   2  exploit/windows/smb/ms17_010_eternalblue       2017-03-14       average  Yes    MS17-010 EternalBlue SMB Remote Windows Kernel Pool Corruption
+   3  exploit/windows/smb/ms17_010_eternalblue_win8  2017-03-14       average  No     MS17-010 EternalBlue SMB Remote Windows Kernel Pool Corruption for Win8+
+   4  exploit/windows/smb/ms17_010_psexec            2017-03-14       normal   Yes    MS17-010 EternalRomance/EternalSynergy/EternalChampion SMB Remote Windows Code Execution
+   5  exploit/windows/smb/smb_doublepulsar_rce       2017-04-14       great    Yes    SMB DOUBLEPULSAR Remote Code Execution
+
+
+Interact with a module by name or index, for example use 5 or use exploit/windows/smb/smb_doublepulsar_rce
+```
+
+Since it's a windows 7 host that's being attacked the second option in the list seemed like a perfect candidate. 
```

Comment:
Since we're attacking a Windows 7 host, the second optiion in the list seemed like a perfect candidate.